working/0723-static-extensions/feature-specification.md

```diff
@@ -126,69 +277,50 @@ extension<X extends Comparable<X>> on SortedList<X> {
 }
 ```
 
-An extension with type parameters can be used to constrain the possible
-type arguments passed to a constructor invocation:
-
-```dart
-extension E4<X> on Map<X, List<X>> {
-  factory Map.listValue(X x) => {x: [x]};
-}
-
-var map = Map.listValue(1); // Inferred as `Map<int, List<int>>`.
-// `Map<int, double>.listValue(...)` is an error.
+An extension can declare factories and redirecting generative
+constructors, but it cannot declare a non-redirecting generative
+constructor.
 
-extension E6<Y> on Map<String, Y> {
-  factory Map.fromString(Y y) => {y.toString(): y};
-}
+Constructors declared in extensions may not be used as
+super-initializers, nor as targets of redirecting generative
+constructors.
```

Comment:
Can they be use as the redirection target of a redirecting factory and, if so, can they be referenced through the on-declaration:
```
class A {
  A.a();
  factory A.b() = E.d; // Through the extension name. 
  factory A.c() = A.d; // Through the on-declaration.
}
extension E on A {
  A.d() : this.a();
}
```
?